src/Content.jsx

```diff
@@ -0,0 +1,24 @@
+export const Part = ({ part, exercises }) => {
```

Comment:
FIX: `Part` must be extracted as a single component in a single file. Then, imported as many times as required in `Content`.